external/implementations/samlconf-keycloak-impl/README.md

```diff
@@ -4,6 +4,8 @@
 * Download Keycloak from [Keycloak Downloads](https://www.keycloak.org/downloads.html).
 * Unzip and run in standalone mode. Then setup an account with "admin" for both username and password. Then login.
 (Steps 2.1-2.4 in the [Keycloak Getting Started Guide](http://www.keycloak.org/docs/latest/getting_started/index.html#booting-the-server)).
+* Go to the Users tab and click the View All Users button.
+* Select the admin user and enter a valid email address under the Email field. Click Save.
```

Comment:
❓ Does it need to be valid, or does it just need to be in the form of an `addr-spec` as defined in [IETF RFC 2822](https://www.rfc-editor.org/info/rfc2822)? Either way, I think this instruction should be qualified to avoid users adding valid emails such as `John Doe <john.doe@mail.com>`, unless you've confirmed that Keycloak is correctly converting those values to the `addr-spec` format.